### PR TITLE
Transition to Transifex

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
**Resolves**

Resolves #27 

**Changes**

Move to Transifex

**Reason for changes**

Use Transifex

**Tests**

This is under testing

---
This was edited by @wgyt and @thecoder876 to include info

### CODE REVIEW

**Review by wgyt — APPROVED** [review]

**Comment by wgyt** [comment]
@GrahamSH-LLK you can merge this if you want

**Comment by GrahamSH-LLK** [comment]
This needs more extensive testing to make sure I didn't break anything

**Review by thecoder876 — APPROVED** [review]
seems fine, ready to merge after testing (marked as draft because it isn't ready yet)

**Comment by wgyt** [comment]
> This needs more extensive testing to make sure I didn't break anything

Go ahead and test it

**Comment by GrahamSH-LLK** [comment]
> > This needs more extensive testing to make sure I didn't break anything
> 
> Go ahead and test it

Go test it yourself, I'm going outside.

**Comment by thecoder876** [comment]
> > > This needs more extensive testing to make sure I didn't break anything
> > 
> > 
> > Go ahead and test it
> 
> Go test it yourself, I'm going outside.

lol

File: localization/strings.en.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-  fatalerror: "✖ 1host.js Error",
-};
+const { t } = require("@transifex/native");
+t("Starting...");
+t("Yikes, we ran into an error running your project\n");
+t("1host.js Message");
+t("✔ 1host.js Success");
+t("⚠ 1host.js Warning");
+t("✖ 1host.js Error");
+t("✖ 1host.js Fatal Error");
```
Comment (thecoder876) [inline localization/strings.en.js:8]:
```suggestion
t("1Host.js Success");
t("1Host.js Warning");
t("1Host.js Error");
t("1Host.js Fatal Error");
```
@GrahamSH-LLK pls remove emojis and capitalize "1host"

**Review by thecoder876 — CHANGES_REQUESTED** [review]

File: localization/strings.en.js (hunk 1 above)
Comment (wgyt) [reply localization/strings.en.js:8]: @thecoder876 this is intended

Comment (GrahamSH-LLK) [reply localization/strings.en.js:8]: A. the emojis are cool
B. it's 1host, not 1Host

Comment (thecoder876) [reply localization/strings.en.js:8]: > [ping removed] this is intended

@wgyt oh ok

**Review by thecoder876 — APPROVED** [review]
approved (again)

**Comment by thecoder876** [comment]
finished with testing?

**Pull request merged**